City/LoginViewController/LoginTextFieldView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  LoginTextFieldView.swift
+//  City
+//
+//  Created by Ivan Zakharov on 25/8/21.
+//
+
+import Foundation
+import UIKit
+import Framezilla
+
+final class LoginTextFieldView: UIView {
+
+    let textField: UITextField = {
+        let textField = UITextField()
+        textField.clearButtonMode = .whileEditing
+        textField.borderStyle = .roundedRect
+        return textField
+    }()
+    let label: UILabel = {
+        let label = UILabel()
+        label.textColor = .red
+        label.font = UIFont(name: "Arial", size: 12)
+        label.isHidden = true
+        return label
+    }()
+    init(placeholderText: String, errorText: String, secureText: Bool) {
```

Comment:
булевые переменные начинаются с глагола, часто это is `isTextSecure`
```suggestion
    init(placeholderText: String, errorText: String, isTextSecure: Bool) {
```